pom.xml

```diff
@@ -3,14 +3,14 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>akka-java</groupId>
     <artifactId>akka-cluster-openshift</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.1-SNAPSHOT</version>
```

Comment:
Does it make sense to use SNAPSHOT here?  Maybe it does.  Just askin'...

Reply:
I chose 1.1-SNAPSHOT so that it was still a minor version upgrade without introducing a GA release. There was no 1.0.0.